frontend/src/components/scheduleEntry/FormScheduleEntryList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="e-form-container">
+    <v-card outlined
+            color="grey lighten-3" class="period mb-2 rounded-b-0">
+      <v-row no-gutters>
+        <v-col>
+          <legend class="pa-2">
+            {{ $tc('components.activity.createScheduleEntries.name') }}
+          </legend>
+        </v-col>
+      </v-row>
+      <form-schedule-entry-item v-for="(scheduleEntry, index) in scheduleEntries"
+                                :key="index"
+                                :schedule-entry="scheduleEntry"
+                                :periods="periods"
+                                :is-last-item="scheduleEntries.length === 1"
+                                @delete="deleteEntry(index)" />
+
+      <v-row>
+        <v-col cols="12" class="text-center">
+          <button-add class="mb-2" @click="addScheduleEntry" />
+        </v-col>
+      </v-row>
+    </v-card>
+  </div>
+</template>
+<script>
+import FormScheduleEntryItem from './FormScheduleEntryItem.vue'
+import ButtonAdd from '@/components/buttons/ButtonAdd.vue'
+
+export default {
+  name: 'CreateActivityScheduleEntries',
+  components: { FormScheduleEntryItem, ButtonAdd },
+  props: {
+    scheduleEntries: {
+      type: Array,
+      required: true
+    },
+
+    // all available periods
+    periods: {
+      type: Array,
+      required: true
+    },
+
+    // currently visible period
+    period: {
+      type: Function,
+      required: true
+    }
+  },
+  data () {
+    return {
+      localScheduleEntries: this.scheduleEntries
+    }
+  },
+  methods: {
+    addScheduleEntry () {
+      this.localScheduleEntries.push({
+        period: () => (this.period)(),
```

Comment:
Was macht das? Warum muss `period` in dieser lokalen Datenstruktur überhaupt eine Funktion sein?

Reply:
Good questions. Gefällt mir eigentlich auch nicht. 

Ich hab die Datenstruktur im Moment übernommen, wie sie in https://github.com/ecamp/ecamp3/blob/devel/frontend/src/components/scheduleEntry/ScheduleEntries.vue#L74 definiert ist. Da will ich im Moment aber nix ändern, bevor #2334 nicht gemerged ist, sonst hagelts merge conflicts.

**Vorschlag:** Ich versuch einen Refactor mit der Implementation von #2350. Da muss ich das sowieso nochmals anfassen.